Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -439,13 +455,17 @@ def get_ip_reputation(ip, threshold=None, status="active,inactive"):
     dbot_context = get_dbot_context(indicator, threshold)
     ip_context = get_ip_context(indicator, threshold)
     threat_ip_context = get_threat_generic_context(indicator)
+    threat_ip_tags = threat_ip_context.pop('Tags', [])
+    if threat_ip_tags:
+        threat_ip_context["Tags"] = ', '.join(str(tag.get('name', '')) for tag in threat_ip_tags)
```

Comment:
Please add a comment explaining that for hr you are displaying the list as a string and then overriding it for the context (in all places).
And let's be consistent with the apostrophes in all places (`'` instead of `"`)
```suggestion
        threat_ip_context['Tags'] = ', '.join(str(tag.get('name', '')) for tag in threat_ip_tags)
```